packages/http-client/src/http-client.ts

```diff
@@ -401,10 +404,16 @@ export function executeHttpRequestWithOrigin<
   T extends HttpRequestConfigWithOrigin
 >(
   destination: DestinationOrFetchOptions,
-  requestConfig: T,
+  requestConfig?: T,
   options?: HttpRequestOptions
 ): Promise<HttpResponse> {
-  return execute(executeWithAxios)(destination, requestConfig, options);
+  const requestConfigWithOrigin =
```

Comment:
[req] This name is not good. Perhaps `requestConfigWithDefaults` - perhaps you can also do a `{...getDefault(),...requestConfig} This would mean the config get all the default values.

Reply:
I guess since the method is mandatory this does not make a difference.

Reply:
I adjusted the variable name to `requestConfigWithDefaults` .
The idea for `requestConfig ?? getDefaultHttpRequestConfigOptions();` was to avoid the getDefault function altogether since the method property is mandatory.